openquake/server/db/actions.py

```diff
@@ -116,21 +114,22 @@ def get_calc_id(datadir, job_id=None):
     calc_id = 0 if not calcs else calcs[-1]
     if job_id is None:
         try:
-            job_id = models.OqJob.objects.latest('id').id
-        except exceptions.ObjectDoesNotExist:
+            job_id = db('SELECT seq FROM sqlite_sequence WHERE name="job"',
```

Comment:
Yes, please check the SQLite documentation. 'SELECT seq FROM sqlite_sequence WHERE name=TABLE' returns the last inserted row ID in the given table.